unityfiles/Assets/Scripts/Game/Collectables/Pickups/DoubleFireRateManager.cs

```diff
@@ -52,22 +56,25 @@ public class DoubleFireRateManager : MonoBehaviour {
     }
 
     private void CallbackWeaponChangedEvent(object _, WeaponChangedEventArgs args) {
-        this.Cleanup ();
+        this.Cleanup();
         this.usedGunController = args.NewGunController;
         this.initialFireRate = args.NewGunController.FireRate;
-        this.UpdateEffectOnController ();
+        this.UpdateEffectOnController();
     }
 
     private void CallbackOnPowerupCollected(object _, PowerUpCollectedArgs args) {
-        if (PICKUP_TYPE.DOUBLE_FIRE_RATE == args.Effect){
+        if (PICKUP_TYPE.DOUBLE_FIRE_RATE == args.Effect) {
             StartCoroutine(Effect(this.secEffectDuration, this.rollingID));
+            var payload = new OnPickupDoubleFireRateArgs();
+            payload.type = type;
+            payload.active = true;
+            OnPickupDoubleFireRateActive?.Invoke(this, payload);
```

Comment:
siden consistency og stil har blitt maset etter, args har blitt brukt i hele resten av kodebasen ikke payload

Reply:
ser den. Men var pga args allerede er brukt.. Så et annet sted i kodebasen at vi har brukt argsss.. Det er jo ikke bedre..

Reply:
args allerede brukt? tror ikke i det idette scopet.
argsss er bare en skrivefeil av meg siden jeg har en utrolig høy keyrepeat pga vim

Reply:
og helt enig ingen av disse er bedre per say men args ar blitt brukt overalt ellers. hvis du ønsker og bytte det till paylod så er det helt ok for meg men da må du også bytte på alle andre eventer i prosjektet

Reply:
Se linje 65

Her er den med argsss. Tilnærmet lik den over, så det var det jeg tok utgangspunkt i:

private void CallbackOnPowerupCollected(object _, PowerUpCollectedArgs args) {
        if (PICKUP_TYPE.LIVE == args.Effect){
            var argsss = new OnPickupLivesUpdateArgs();
            argsss.LivesToAdd = livesValue;
            OnPickupLivesUpdate?.Invoke(this, argsss);
        }
    }

Reply:
For ordens skyld: Jeg liker args helt fint.